src/main/java/library/app/dao/RentDaoImpl.java

```diff
@@ -31,7 +31,14 @@ public void add(Rent rent) {
                 .createQuery("FROM Rent WHERE user_id=:user AND book_id=:book");
         query.setParameter("user", user.getId());
         query.setParameter("book", book.getId());
-        return Optional.ofNullable((Rent) query.getSingleResult());
+        try {
```

Comment:
should we use try-catch here?

Reply:
Sorry, it was for debug. Will be deleted in a moment